waspc/test/SemanticVersionTest.hs

```diff
@@ -0,0 +1,39 @@
+module SemanticVersionTest where
+
+import Test.Tasty.Hspec
+import Wasp.SemanticVersion
+
+spec_isVersionInBounds :: Spec
+spec_isVersionInBounds = do
+  it "Correctly determines satisfied exact version bounds" $
```

Comment:
@Martinsos How do I decide whether to put a "describe" or not?

Reply:
Well, if you have a bunch of `it`s and you want to group them, then you use `describe`!
Imagine you see these tests failing -> if you would love to see some extra context for all of them, then you do a describe. Or if you have some code that they all need to share.
It is just a hierarchy thing -> "describe" are nodes, "it" is leaves.